resources/views/dashboard/index.blade.php

```diff
@@ -25,7 +25,7 @@
                             <form class='component-inline form-vertical' data-messenger="{{trans('dashboard.components.edit.success')}}">
                                 <div class="row striped-list-item">
                                     <div class="col-lg-4 col-md-3 col-sm-12">
-                                        <h4>{{ $component->name }}</h4>
+                                        <h4>{{ $component->name }} {{ $component->displayVersion() }}</h4>
```

Comment:
Once the `displayVersion` method is in the presenter, you'll need to replace this with just `$component->display_version`.

Better yet, create a new localization line for it, and you can have it in there, formatted however you want.